rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,37 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+All dxw-owned respositories MUST use a standard linting tool for the
```

Comment:
All MUST? That's a bit extreme. Who is going to go and add linters to all the archived repos? What about all the private repos that are just little shell scripts? We have repos containing shell scripts that were only used once.

Reply:
This summary contradicts the proposal section:

> Where available, all new dxw-owned or maintained repositories MUST use a

I suggest removing the RFC 2119 wording from the summary (and also updating it to reflect reality).

Reply:
I think, in my head, I was referring to all new repos, but you're right, it's probably a bit extreme for tiny shell scripts or one-shot projects.